Source/Orts.Simulation/Simulation/Turntables.cs

```diff
@@ -81,6 +81,11 @@ public class MovingTable
         protected int[] MyTrVectorSectionsIndex;
         public bool[] MyTrackNodesOrientation { get; protected set; } // true if forward, false if backward
         public int TrackShapeIndex;
+        public enum submessagecode
```

Comment:
This should be in CamelCase.

Reply:
Since this an enum type definition, this should be UpperCamelCase, not lowerCamelCase.

Reply:
> Since this an enum type definition, this should be UpperCamelCase, not lowerCamelCase.

Done.